django_cache_manager/mixins.py

```diff
@@ -95,8 +98,9 @@ def cache_backend(self):
         """
         if not hasattr(self, '_cache_backend'):
             # determine django version for getting cache backend
-            if django.get_version() >= '1.7':
```

Comment:
It is, but the versions can't be compared as strings, because e.g. `'1.11' < '1.7' == True`